### ISSUE TITLE
Already staked token it's show in dashboard need to stake

### ISSUE THREAD

**Issue by Balasuriya55** [Shardeum/shardeum-bug-reporting#411]
![Screenshot_2023-06-20-08-04-07-480_site mises browser](https://github.com/Shardeum/shardeum-bug-reporting/assets/120310798/db1be10e-4282-4309-a3ca-eef7a4c217eb)
##### What is the issue?

:red_circle:  Describe the error in simple terms. :red_circle: 
    
#### What impact does the issue have?

:red_circle: Describe if the bug is a visual warning or if it breaks functionality causing a system to fail. :red_circle: 

##### How to reproduce the issue? 

:red_circle: 
Explain in as much detail as possible to help reproduce the bug:
```
    -where the error is seen
    -how the error was potentially caused if possible (mention previous steps taken to see the error if relevant)
    -share code and explain how the code was run
```
:red_circle: 

##### What other resources can you share regarding this issue?

:red_circle: Photos and links related to the error. :red_circle:

**Comment by Balasuriya55** [comment]
I have facing this issue already staked and activated my node but now dash board show need to stake

**Comment by ajbaba** [comment]
![Screenshot_20230622-230335](https://github.com/Shardeum/shardeum-bug-reporting/assets/111224305/0348e99e-ce26-471d-a5e4-a45e2977a1c0)
This has been my experience since last time my node validated. It's showing need stake despite the fact that I have 12SHM staked. How do I get rid of that? I have tried resetting MM but the result stays the same. Meanwhile, staking and unstaking aren't working also. Testnets should be usable by now.

**Comment by swarna1101** [comment]
update to the latest node its solved in latest version 1.9.5.
